pkg/pillar/types/assignableadapters.go

```diff
@@ -192,7 +212,12 @@ const (
 	IoNetWLAN IoType = 5
 	IoNetWWAN IoType = 6
 	IoHDMI    IoType = 7
-	IoOther   IoType = 255
+	// enum 8 is reserved for backward compatibility with controller API
+	PhyIoNVMEStorage IoType = 9
+	PhyIoSATAStorage IoType = 10
+	PhyIoNetEthPF    IoType = 11
+	PhyIoNetEthVF    IoType = 12
```

Comment:
For these also keep the same prefix ("Io and IoNet") as for the existing values.